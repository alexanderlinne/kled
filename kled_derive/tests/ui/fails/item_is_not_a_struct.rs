#[macro_use]
extern crate kled_derive;

#[operator(type = "flow")]
pub enum Map {}

fn main() {}
