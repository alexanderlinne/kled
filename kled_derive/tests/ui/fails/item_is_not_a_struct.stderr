error: expected `struct`
 --> $DIR/item_is_not_a_struct.rs:5:5
  |
5 | pub enum Map {}
  |     ^^^^
